Source/Fuse.Reactive.Expressions/MathFunctions.uno

```diff
@@ -425,10 +425,22 @@ namespace Fuse.Reactive
 			float4 av = float4(0), bv = float4(0);
 			int asize = 0, bsize = 0;
 			float tv = 0;
-			if (!Marshal.TryToZeroFloat4(a, out av, out asize) ||	
-				!Marshal.TryToZeroFloat4(b, out bv, out bsize) ||
-				!Marshal.TryToType<float>(t, out tv))
+
+			var aValid = Marshal.TryToZeroFloat4(a, out av, out asize);
+			var bValid = Marshal.TryToZeroFloat4(b, out bv, out bsize);
+			var tValid = Marshal.TryToType<float>(t, out tv);
+			if (!aValid)
+				Fuse.Diagnostics.UserWarning("The first argument of lerp is not supported.", this);
+			if (!bValid)
+				Fuse.Diagnostics.UserWarning("The second argument of lerp is not supported.", this);
+			if (!tValid)
+				Fuse.Diagnostics.UserWarning("The third argument of lerp is not supported.", this);
```

Comment:
Actually, I suspect that this should use `ReportTemporalUserWarning` instead, and I think those are temporarily ignored...

Reply:
I think `UserWarning` is fine, since this function won't be called until all values are available. It's not a "temporal" warning like the "doesn't exist" warnings are. Though I would argue it's a `UserError` here, not a warning. They're provdiing invalid values to lerp.

Reply:
OK, if that's the case, I agree.